yarn.lock

```diff
@@ -2389,6 +2389,11 @@ anymatch@^2.0.0:
     micromatch "^3.1.4"
     normalize-path "^2.1.1"
 
+app-root-path@^2.2.1:
```

Comment:
I don't believe you should make any changes to this file (you didn't add any new external dependencies).

Feel free to revert.